airflow/www/views.py

```diff
@@ -217,6 +217,32 @@ def get_safe_url(url):
     return redirect_url.geturl()
 
 
+def build_scarf_url(dags_count: int) -> str:
+    """Build the URL for the Scarf telemetry collection."""
+    if not settings.is_telemetry_collection_enabled():
+        return ""
+
+    scarf_domain = "https://apacheairflow.gateway.scarf.sh"
+
+    platform_sys, platform_arch = scarf.get_platform_info()
+    db_version = scarf.get_database_version()
+    db_name = scarf.get_database_name()
```

Comment:
```suggestion
    db_engine_name = scarf.get_database_engine_name()
```
if the above comment is considered

Reply:
I think that's ok -- it is just for our usage and not external users